src/api/spec/models/groups_user_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe GroupsUser do
+  let(:group) { create(:group) }
+  let(:user) { create(:confirmed_user, login: 'eisendieter') }
+
+  it 'creates an AddedToGroupEvent when a user is added to a group' do
+    allow(Event::AddedUserToGroup).to receive(:create)
+
+    group.add_user(user)
+
+    expect(Event::AddedUserToGroup).to have_received(:create).with(group: group, user: user)
```

Comment:
Hm, how do you expect this to break? 🤔

Reply:
Well, the call is behaviour.  And the call is made from `#add_user`.  The mock is about testing the edges, not the nodes.